skipruntime-ts/ts/src/package.json

```diff
@@ -1,15 +1,10 @@
 {
   "type": "module",
   "scripts": {
-    "lint": "eslint ."
+    "lint": "eslint --config ../../../eslint.config.js ."
   },
   "devDependencies": {
-    "@eslint/js": "^9.9.1",
-    "@stylistic/eslint-plugin-js": "^2.6.4",
-    "@types/node": "^22.5.5",
-    "@typescript-eslint/parser": "^8.3.0",
-    "eslint-plugin-jsdoc": "^50.2.2",
-    "prettier": "3.2.5",
-    "typescript-eslint": "^8.3.0"
+    "@eslint/js": "",
+    "@types/node": ""
   }
```

Comment:
I think you can completely remove the `devDependencies` from modules if they are defined at the workspace root

Reply:
I can remove eslint/js, but something re-adds types/node by itself.